src/pipecat/services/riva.py

```diff
@@ -35,6 +35,7 @@
     )
     raise Exception(f"Missing module: {e}")
 
+FASTPITCH_TIMEOUT = 5
```

Comment:
is this seconds? milliseconds?

Reply:
Seconds :smiley:  . Renamed to `FASTPITCH_TIMEOUT_SECS`